server/lib/pbench/indexer.py

```diff
@@ -3109,7 +3109,7 @@ def hostnames_if_ip_from_sosreport(sos_file_name):
         try:
             hostname_f = str(sostb.extractfile(hostname_f_file[0]).read(), 'iso8859-1')[:-1]
         except IOError as e:
-            raise SosreportHostname("Failure to fetch a hostname-f from the sosreport")
+            return ('error',"Failure to fetch a hostname-f from the sosreport")
```

Comment:
The return code part should not be a string: you can always define an [enumeration](https://docs.python.org/3/library/enum.html?highlight=enumeration) for the various  values and return that instead.